src/Builder/ArchiveBuilder.php

```diff
@@ -113,11 +113,11 @@ public function dump(array $packages)
                     ;
 
                     if (!file_exists($path)) {
-                        $downloadDir = sys_get_temp_dir() . '/composer_archiver/' . $packageName;
+                        $downloadDir = sys_get_temp_dir().'/composer_archiver/'.$packageName;
```

Comment:
I think this does not correspond to psr-2 [https://github.com/php-fig-rectified/fig-rectified-standards/blob/master/PSR-2-R-coding-style-guide-additions.md#strings-and-concatenation](url)

Reply:
I guess I used the wrong rule. Gonna change it